backend/test_observer/external_apis/c3/c3.py

```diff
@@ -0,0 +1,48 @@
+import os
+from collections.abc import Iterable
+
+import requests
+from requests import Request
+
+from .models import SubmissionStatus
+
+
+class C3Api:
+    def __init__(self, client_id: str | None = None, client_secret: str | None = None):
+        if not client_id:
+            client_id = os.environ["C3_CLIENT_ID"]
+        if not client_secret:
+            client_secret = os.environ["C3_CLIENT_SECRET"]
+
+        self.client_id = client_id
+        self.client_secret = client_secret
+
+        self.bearer_token = ""
+
+    def get_submissions_statuses(self, ids: Iterable[int]):
+        session = requests.session()
+        request = Request(
+            method="GET",
+            url="https://certification.canonical.com/api/v2/submissions/status/",
```

Comment:
Should we update the url to be more dynamic, as we might want to work in C3 and TO development at the same time?

Reply:
Good point. It doesn't sound good for dev deployment to call C3 production. But I don't want this to break in development either. So perhaps we could actually create another class altogether that implements the same public methods as this `C3` class. Except that class would just return say empty objects (cause it's for development). Then we'd use FastAPI's dependency injection to use the appropriate class depending on the environment. This would also mean that we need to pass the environment to the backend application.